hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2238,17 +2238,35 @@
   <property>
     <name>hdds.key.len</name>
     <value>2048</value>
-    <tag>SCM, HDDS, X509, SECURITY</tag>
+    <tag>SCM, HDDS, X509, SECURITY, CRYPTO_COMPLIANCE</tag>
+    <description>
+      SCM CA key length. This is an algorithm-specific metric, such as modulus
+      length, specified in number of bits.
+    </description>
+  </property>
+  <property>
+    <name>hdds.key.algo</name>
+    <value>RSA</value>
+    <tag>SCM, HDDS, X509, SECURITY, CRYPTO_COMPLIANCE</tag>
+    <description>
+      SCM CA key algorithm.
+    </description>
+  </property>
+  <property>
+    <name>hdds.security.provider</name>
+    <value>BC</value>
+    <tag>OZONE, HDDS, X509, SECURITY, CRYPTO_COMPLIANCE</tag>
     <description>
-      SCM CA key length.  This is an algorithm-specific metric, such as modulus length, specified in number of bits.
+      The main security provider used for various cryptographic algorithms.
     </description>
   </property>
   <property>
     <name>hdds.key.dir.name</name>
     <value>keys</value>
     <tag>SCM, HDDS, X509, SECURITY</tag>
     <description>
-      Directory to store public/private key for SCM CA. This is relative to ozone/hdds meteadata dir.
+      Directory to store public/private key for SCM CA. This is relative to
+      ozone/hdds meteadata dir.
```

Comment:
Could you remove this? I don't this it is related to this change, you only broke the line into two lines.